CMakeLists.txt

```diff
@@ -15,15 +15,19 @@
 cmake_minimum_required(VERSION 2.8.9)
 project(EtcTest)
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -I/usr/include/i386-linux-gnu/c++/4.8 -I/usr/include/c++/4.8")
+
 set (CMAKE_CXX_STANDARD 11)
 IF (APPLE)
-	set (CMAKE_CXX_FLAGS "-I/usr/include/i386-linux-gnu/c++/4.8 -I/usr/include/c++/4.8 -std=c++11 -g3 -Wall -O3")

```

Comment:
Both CMAKE_CXX_STANDARD and CXX_STANDARD_REQUIRED require min cmake 3.1 so cmake_minimum_required should be bumped too.